src/NServiceBus.AcceptanceTesting/NServiceBus.AcceptanceTesting.csproj

```diff
@@ -66,6 +66,7 @@
     <Compile Include="Support\EndpointConfiguration.cs" />
     <Compile Include="Support\EndpointRunner.cs" />
     <Compile Include="Support\FailTestOnErrorMessageFeature.cs" />
+    <Compile Include="Support\IConfigureTestEnvironment.cs" />
```

Comment:
Probably nitpicking here, but doesn't this allow to configure test run rather than environment?
`IConfigureTestRun` or `IConfigureTestExecution`